hw/ip/spi_device/doc/_index.md

```diff
@@ -19,9 +19,22 @@ title: "SPI Device HWIP Technical Specification"
 - Interrupts for RX/TX SRAM FIFO conditions (empty, full, designated level for
   RX, TX)
 
+### TPM over SPI
+
+- up to 64B compile-time configurable read and write data buffer (default: 4B)
+- 1 TPM command (8b) and 1 address (24bit) buffer
+- HW controlled wait state
+- Shared SPI with other SPI Device functionalities. Unique CS# for the TPM
+- HW processed registers for the read requests
+  - TPM_ACCESS_x, TPM_STS_x, TPM_INTF_CAPABILITY, TPM_INT_ENABLE, TPM_INT_STATUS, TPM_INT_VECTOR, TPM_DID_VID, TPM_RID
+  - TPM_HASH_START returns FFh
```

Comment:
do you think it's at all worth mentioning the bit on the fifo interface and the other interface? (can't remember what it's called now)

Reply:
o could you link the publicly available doc here also?